tracker/src/Tracker/Syntax/Option.hs

```diff
@@ -0,0 +1,12 @@
+module Tracker.Syntax.Option
+  ( unNone
+  ) where
+
+import RIO
+
+unNone :: [a] -> [Maybe a] -> [a]
```

Comment:
Why not make a function with a simpler signature and default acc=`[]`?

Reply:
`unNone :: [Maybe a] -> [a]`

Reply:
simplified